docs/pages/configuration/pipelines/functions/run_dependency_pipelines.mdx

```diff
@@ -2,3 +2,37 @@
 title: Function run_dependency_pipelines
 sidebar_label: run_dependency_pipelines
 ---
+
+Runs dependencies for pipelines.
+
+## Arguments
+
+### `--pipeline`
+
+The pipeline to be deployed from the dependency.
+
+### `--exclude`
+
+The dependencies to be excluded from the pipeline.
+
+### `--only`
+
+The dependencies to be included in the pipeline.
+
+## Usage
+
+```
+run_dependency_pipelines [buildDependencies] [renderDependencies]
```

Comment:
I don't really understand this usage section.

What options do I have for the buildDependencies arg or renderDependencies arg? The usage in `run_dependency_pipelines.go` is `run_dependency_pipelines dep1 dep2` and `run_dependency_pipelines --all`.

Reply:
Yes exactly, dependencies shouldn't be grouped by build or render, rather they are independent DevSpace projects that you want to run a pipeline from completely.